README.md

```diff
@@ -56,186 +24,143 @@ For more information on the importance of a professional README for your reposit
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">
+            <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/ruby/ruby-plain-wordmark.svg" alt="Ruby" width="55" height="55" />
+          </a></li>
+    <li><a href="https://rspec.info/">
+            <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/rspec/rspec-original-wordmark.svg" alt="Rspec" width="55" height="55" />
+          </a></li>
   </ul>
 </details>
 
 <details>
 <summary>Database</summary>
   <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://www.postgresql.org/"><img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/postgresql/postgresql-plain-wordmark.svg" alt="PostgreSQL" width="55" height="55" /></a></li>
   </ul>
 </details>
 
-<!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **E-nventorium allows users to catalog their digital assets, such as music albums, movies, e-books, software, and more.**
+- **Users can create new catalog items by providing relevant information, such as title, author/creator, genre, source, label, and publish date.**
+- **The application includes functionality to archive items based on specific criteria, such as publish date. Items can be moved to an archived state, marking them as no longer in active use.**
+- **Users can manage genres and labels associated with their items. Genres and labels can be predefined or created on the fly during item creation.**
+- **The application stores cataloged items and related data persistently, so users can access their catalog even after exiting the application.**
+- **E-nventorium operates through a Command Line Interface (CLI), allowing users to interact with the application through text-based commands and responses.**
+- **The CLI provides intuitive commands for creating items, searching, archiving, and managing genres/labels, making the application user-friendly even for those familiar with basic command-line usage.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+## 📼 Video Demo <a name="video-demo"></a>
 
-- [Live Demo Link](https://google.com)
+You can watch a video demonstration of this project [here](https://drive.google.com/file/d/1IEowRCY40RH0eyC43MlsrfzWzLr8nW6Y/view?usp=drive_link).
```

Comment:
- [ ] Hi friends, I checked on the video presentation and found out that the length is 7mn 33s. I would suggest that you stick to the requirement: I would advise you to record again and not exceed 5 mn as required.
- [ ]  **My suggestion** is that you should allow only one person to make the presentation of the App, and the rest of you can just introduce themself(Mention the name) and follow the presenter. Your camera should be ON. This will help you to save time. Also, do not run `Rspec test` in the video to save time. 
All the best 👍

Reply:
@OLIPLICHE Thank you, we will do as you asked 😌